shopfloor_reception/migrations/14.0.2.1.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from openupgradelib import openupgrade
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+def move_fields_to_new_module(cr):
+    # is_shopfloor_created has been wrongly put in shopfloor_reception.
+    # this script moves it in shopfloor.
+    openupgrade.update_module_moved_fields(
+        cr, "stock.picking", "is_shopfloor_created", "shopfloor_reception", "shopfloor"
+    )
```

Comment:
Does it work well if `shopfloor` is upgraded first (which will create the `stock_picking.is_shopfloor_created` column in DB), then comes the upgrade of `shopfloor_reception` with this script?